frontend/ui/widget/spinwidget.lua

```diff
@@ -64,8 +66,8 @@ function SpinWidget:update()
         value = self.value,
         value_min = self.value_min,
         value_max = self.value_max,
-        value_step = 1,
-        value_hold_step = 4,
+        value_step = self.value_step,
+        value_hold_step = self.value_hole_step,
```

Comment:
I think you meant hol**d** on both sides :) (here and  there's other **hole**s elsewhere)

Reply:
Fixed :) Thanks.